WcaOnRails/spec/lib/delayed/plugins/save_completed_jobs_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+class SuccessfulJob < ActiveJob::Base
+  def perform
+    puts "Succeeding!"
+  end
+end
+
+class FailingJob < ActiveJob::Base
+  def perform
+    raise "Failure!"
+  end
+end
+
+describe Delayed::Plugins::SaveCompletedJobs, type: :feature do
```

Comment:
This is meant to be a test of the `Delayed::Plugins::SaveCompletedJobs` plugin I wrote, which does 2 things: it saves completed jobs in the `completed_jobs` table, and it notifies us via email when a job fails.